contrib/diurnal/example-diurnal-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: diurnalcontroller
+  name: diurnalcontroller
+spec:
+  replicas: 1
+  selector:
+    name: diurnalcontroller
+  template:
+    metadata:
+      labels:
+        name: diurnalcontroller
+    spec:
+      containers:
+        - command: ["diurnal_math", "-ns", "default", "-labels", "name=redis-slave", "-sample", "300", "-countfn", "10+10*sin(t)"]
```

Comment:
I thought it was an interesting demonstration to allow arbitrary functions here, but maybe it needs a better name?  It might be that the really useful functionality is just [](time, value) pairs and good (non-linear?) interpolation between them